### ISSUE TITLE
Bug: Pasting to custom cell causes panic

### ISSUE THREAD

**Issue by infinitepr0** [glideapps/glide-data-grid#274]
Here's my implementation:
```js
// cell
return {
  kind: GridCellKind.Custom,
  allowOverlay: true,
  data: data,
  copyData: JSON.stringify(data),
}

// onCellEdited
(cell, value) => {
  ...
  const parse = (data) => {return JSON.parse(data)}
  ...
  setValueOfCell(parse(value))
}
```

This should work. However, paste support has been explicitly disabled for custom cells.
```
[Error] Unhandled Promise Rejection: Error: Hell froze over
	panic (support.js:12)
	pasteToCell (data-editor.js:1621)
	(anonymous function) (data-editor.js:1674)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise
```

**Comment by jassmith** [comment]
ah woops seems there is a bug in the types somewhere, hunting for it.

**Comment by jassmith** [comment]
Fixed in 4.0.3
